### ISSUE TITLE
Please add a simple checkbox option to disable button magnification

### ISSUE THREAD

**Issue by enomat** [hrydgard/ppsspp#12987]
The button magnification is annoying and distracting. This screenshot is an example. The pressed buttons cover a large part of the screen and cover the game HUD:

![Screenshot_2020-05-30-19-24-32](https://user-images.githubusercontent.com/12612250/83335636-69b99280-a2ae-11ea-8ea2-3d92a407dbe9.jpg)

Not only the large size, but also the act of changing the size is distracting during gaming (because it means extra movement/animation/content change on the screen, and it happens every time a button is pressed).

For example ePSXe has an option to disable this feature: "Buttons Magnifying on press". It is really necessary in PPSSPP.

**Comment by iota97** [comment]
You can change `Button style` to `Glowing borders` in the touch control section of control setting to achieve that.

**Comment by enomat** [comment]
Thank you. That's good.

Before posting this issue I looked through the menu, all the options, but the reason why I didn't try out Button style or Glowing borders is because I remembered that a few years ago I checked out the Button style option, there were 2 options (Thin borders/Classic) and they changed only the skin, the appearance of buttons. I thought it's still the same case, and it would not affect the button magnification. There is no hint that `Glowing borders` disables button magnification/enlargement on press.
